app/views/practices/form/metrics.html.erb

```diff
@@ -0,0 +1,245 @@
+                <td align="left" style="font-weight: bold">Adoptions</td>
+              </tr>
+              <tr class="border_bottom">
+                <td style="width: 80%">1a (High Complexity)</td>
+                <td><%= @a_high_complexity_at %></td>
+              </tr>
+              <tr class="border_bottom">
+                <td style="width: 80%">1b (High Complexity)</td>
+                <td><%= @b_high_complexity_at %></td>
+              </tr>
+              <tr class="border_bottom">
+                <td style="width: 80%">1c (High Complexity)</td>
+                <td><%= @c_high_complexity_at %></td>
+              </tr>
+              <tr class="border_bottom">
+                <td style="width: 80%">2 (Medium Complexity)</td>
+                <td><%= @medium_complexity_2_at %></td>
+              </tr>
+              <tr class="border_bottom">
+                <td style="width: 80%">3 (Low Complexity)</td>
+                <td><%= @low_complexity_3_at %></td>
+              </tr>
+            </table>
+          </div>
+       </div>
+
+<!-- LEADER BOARD -------------------------------------------------------------------------------------------->
+        <div class="margin-bottom-3">
+          <table style="width: 100%">
+            <tr>
+              <td style="font-size: 32px">Leader board</td>
+              <td align="right"><button style="alignment: right; margin: 0px" id="toggle_leader_board_view_30" class="usa-button usa-button--base usa-button--active">Last 30 days</button><button style="alignment: left;  width: 101px; height: 40px; color: black" id="toggle_leader_board_view_all_time" class="usa-button usa-button--outline usa-button--active">All time</button></td>
+            </tr>
+          </table>
+        </div>
+
+        <div id="leader_board_page_views_30_days" class="margin-bottom-3">
+          <table style="width: 100%">
+            <tr class="border_bottom">
+            <td align="left" style="width: 80%; font-weight: bold">Practice</td>
+            <td align="left" style="font-weight: bold">Views</td>
+            </tr>
+            <% @page_views_leader_board_30_days.each do |rec| %>
+              <tr class="border_bottom">
+                <td style="width: 80%"><a target="_blank" href="/practices/<%= rec[:practice_slug] %>/"><%= rec[:practice_name] %></a></td>
+                <td><%= rec[:count] %></td>
+              </tr>
+            <% end %>
+          </table>
+        </div>
+        <div class="margin-bottom-3" id="leader_board_page_views_all_time" style="display: none">
+          <table style="width: 100%">
+            <th align="left" style="width: 80%">Practice</th>
+            <th align="left">Views</th>
+            <% @page_views_leader_board_all_time.each do |rec| %>
+              <tr class="border_bottom">
+                <td style="width: 80%"><a target="_blank" href="/practices/<%= rec[:practice_slug] %>/"><%= rec[:practice_name] %></a></td>
```

Comment:
for these you would want to add the `dm-internal-link` css class to the `<a>`

Reply:
done